### PR TITLE
Bump fmt to 11.0.2 to be equivalent to nrn

### PR DESCRIPTION
**Pull request by alkino**
I blindly try to fix https://bbpgitlab.epfl.ch/hpc/cellular/nrn/-/jobs/1431896
I cannot compile libsonata locally btw

### CODE REVIEW

**Comment by mgeplf** [comment]
> I cannot compile libsonata locally btw

What is the error you get?

**Comment by alkino** [comment]
The error is:
`/usr/bin/ld : libsonata.so.0.1.28 : référence indéfinie vers « void nonstd::variants::detail::index_tag<7ul>(nonstd::variants::detail::index_tag_t<7ul>) »`

I look  everywhere with `nm` the symbol is always `Undefined`

**Comment by mgeplf** [comment]
Can you include the compiler version, etc?
Have you tried using a later release of https://github.com/martinmoene/variant-lite/?

File: MANIFEST.in [hunk 1]
```diff
@@ -8,11 +8,7 @@ include extlib/CMakeLists.txt
 include extlib/nlohmann/json.hpp
 
 # fmt
-recursive-include extlib/fmt/include *
-recursive-include extlib/fmt/src *
-recursive-include extlib/fmt/support/cmake *.cmake
-include extlib/fmt/CMakeLists.txt
-include extlib/fmt/*.rst
+recursive-include extlib/fmt *
```
Comment (mgeplf) [inline MANIFEST.in:11]: why do we have to include things like `extlib/fmt/test/`?
Including more made the sdist size increase by almost 50%.

Comment (alkino) [reply MANIFEST.in:11]: Try to reduce it.

**Review by jorblancoa — APPROVED** [review]

**Pull request merged**